dashboard/src/reducers/cluster.ts

```diff
@@ -146,20 +146,19 @@ const clusterReducer = (
       }
       break;
     case getType(actions.config.receiveConfig):
-      // Initialize the additional clusters when receiving the config.
-      const clusters = {
-        ...state.clusters,
-      };
+      // Initialize the clusters when receiving the config.
       const config = action.payload as IConfig;
+      const clusters: IClustersMap = {};
       config.clusters?.forEach(cluster => {
```

Comment:
so `clusters` should be defined, right? `config.clusters.forEach`

Reply:
Yep, indeed.